tasks/carto.py

```diff
@@ -490,6 +500,10 @@ class OBSMeta(Task):
         AND numer_ctag.column_id = numer_c.id
         AND numer_ctag.tag_id = numer_tag.id
         AND numer_c.id = leftjoined_denoms.all_numer_id
+        AND NOT EXISTS (
+          SELECT 1 FROM agg_wo_universe u
+          WHERE u.id = numer_c.id
+        )
```

Comment:
I have also tested the possible loss of performance of adding this check and it's insignificant

Reply:
Nice to see. Not sure if we want to keep this in this query as it is redundant with the tasks check. My concern is not performance but readability, my head almost exploded reading this. If you prefer to keep it, let's add some comments, at least.